Specs/Scene/ModelExperimental/ModelExperimental3DTileContentSpec.js

```diff
@@ -13,7 +14,7 @@ describe("Scene/ModelExperimental/ModelExperimental3DTileContent", function () {
   const gltfContentUrl = "./Data/Cesium3DTiles/GltfContent/glTF/tileset.json";
   const glbContentUrl = "./Data/Cesium3DTiles/GltfContent/glb/tileset.json";
   const buildingsMetadataUrl =
-    "./Data/Cesium3DTiles/Metadata/FeatureMetadata/tileset.json";
+    "./Data/Cesium3DTiles/Metadata/StructuralMetadata/tileset_1.1.json";
```

Comment:
It doesn't seem like we use the 1.0 version of the StructuralMetadata tileset... do we really need both versions?

Reply:
Ah yeah, that was one bit I included just in case, but probably better to remove the 1.0 version. I can do that.